.nvmrc

```
@@ -1 +1 @@
-16.16.0
+18.16.0
```

```suggestion
18.16.1
```

Corresponds to current Node.js LTS 
https://nodejs.org/en